test/util/genesis/accounts.go

```diff
@@ -20,6 +20,12 @@ type Account struct {
 	InitialTokens int64
 }
 
+var (
+	Utia = sdk.NewCoins(sdk.NewCoin(app.BondDenom, sdk.NewInt(1)))
+	// Tia is 1e6 utia (a.k.a one million utia)
+	Tia = sdk.NewCoins(sdk.NewCoin(app.BondDenom, sdk.NewInt(1e6)))
+)
```

Comment:
[discussion] Since these variables depend on app.BondDenom, I wonder if it makes more sense to export them from app. Potential location after [this block](https://github.com/rootulp/celestia-app/blob/20d2ff2b6dc2a0008535858970cb0f061aeebe89/app/app.go#L101-L110).

Reply:
Not sure at the moment, but I have opinions on it, that
1. defining Utia and tia variables in accounts.go makes sense as it is the center to use them, and it doesn't make the calling tougher.
2. if we have a look at `app.BondDenom` and as we are trying to export them, that's the only way we have at the moment(from my understanding)(just had surface understanding at the moment).

Reply:
> [discussion] Since these variables depend on app.BondDenom, I wonder if it makes more sense to export them from app. Potential location after [this block](https://github.com/rootulp/celestia-app/blob/20d2ff2b6dc2a0008535858970cb0f061aeebe89/app/app.go#L101-L110).

Hey, @rootulp sorry for being too silent on this, I was a bit away from things, yup back working on this, I don't think my above comment makes any sense(my bad on it!), but I got the point you raised and found another good way to set the variables in `global_consts.go` file, which makes the overall variable useful and not conflicting anywhere.

And of the point of using the variables that are dependent on app.BondDenom, it also gets solved when we are storing the variables in `global_consts.go` file, by using directly BondDenom instead of app.BondDenom, does that make sense?

Reply:
hmm `global_consts.go` could be a viable location. Do other reviewers have thoughts on this?